Training5/src/task1/src/components/Filter.js

```diff
@@ -37,32 +37,41 @@ const FilterItem = styled.p`
 
 export default function Filter() {
   const dispatch = useDispatch();
-  const filter = useSelector((state) => state.todo.filter);
-  console.log("filter", filter);
+  const filter = useSelector((state) => state.todo.isLoadCompleted);
 
-  function handleAllClick(){
-    const action = FilterAll();
+  function handleAllClick() {
+    const action = filterAll();
     dispatch(action);
   }
 
-  function handleCompletedClick(){
-    const action = FilterCompleted();
+  function handleCompletedClick() {
+    const action = filterCompleted();
```

Comment:
don't duplicate the action createor for each value of completed, here you can create ONE action creator

const changeFilterValue = (filterValue) => ({
  type: CHANGE_FILTER_VALUE,
  payload: {
  filterValue.
  }
})

and use in everywhere changeFilterValue(true), changeFilterValue(false), changeFilterValue(null)